### PR TITLE
Update IP addresses data

### PR DESCRIPTION
**Pull request by ameukam**
This is done to include CIDRs for newest AWS and GCP regions where registry.k8s.io will be deployed.

### CODE REVIEW

**Comment by ameukam** [comment]
cc @BenTheElder

File: cmd/archeio/internal/app/buckets_test.go [hunk 1]
```diff
@@ -27,9 +27,9 @@ func TestRegionToAWSRegionToHostURL(t *testing.T) {
 	// ensure known regions return a configured bucket
 	regions := []string{}
 	for _, ipInfo := range cloudcidrs.AllIPInfos() {
-		// AWS regions, excluding "GLOBAL" meta region and govcloud
+		// AWS regions, excluding "GLOBAL" meta region, AWS US Gov Cloud and European Soveign Cloud
```
Comment (BenTheElder) [inline cmd/archeio/internal/app/buckets_test.go:30]: FWIW: The name is AWS GovCloud (no spaces) and they usually drop the `(US)` from the end but if it is included its `AWS GovCloud (US)` /shrug

**Review by BenTheElder — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**